### ISSUE TITLE
Angular 6 Not Compatible

### ISSUE THREAD

**Issue by garysui** [bleenco/ng2-datepicker#316]
Due to of version of rxjs of dependency "ngx-slimscroll", ng2-datepicker won't work with Angular 6 which uses rxjs 6. Rxjs 6 has breaking changes.

**Comment by jkuri** [comment]
@diegoalex I did not get your response what's your issue running this on Angular 6?

**Comment by jagabs** [comment]
not working for me

**Comment by kogden** [comment]
I agree. Something opens but none of the days/months/years are showing.

**Comment by mcsekar12** [comment]
Any updates on this issue? Facing the same issue when migrating from angular4 to angular6 .
